src/app/components/InlineLink/index.test.tsx

```diff
@@ -116,4 +117,84 @@ describe('InlineLink', () => {
       );
     },
   );
+
+  it.each`
+    size          | expected
+    ${'atlas'}    | ${'4.875'}
+    ${'elephant'} | ${'3.75'}
+    ${'imperial'} | ${'3.125'}
+    ${'royal'}    | ${'2.5'}
+    ${'foolscap'} | ${'2'}
+    ${'canon'}    | ${'1.75'}
+  `('should apply provided font size ', ({ size, expected }) => {
+    render(
+      <InlineLink
+        to="/mundo/articles/ce42wzqr2mko"
+        text="Hello World!"
+        size={size}
+      />,
+    );
+
+    expect(screen.getByText('Hello World!')).toHaveStyle({
+      color: '#222222',
+      'font-size': `${expected}rem`,
+    });
+  });
+
+  it.each`
+    variant                | fontWeight | fontStyle
+    ${'sansRegularItalic'} | ${400}     | ${'italic'}
+    ${'sansBold'}          | ${700}     | ${'normal'}
+    ${'serifMediumItalic'} | ${500}     | ${'italic'}
+    ${'serifLight'}        | ${300}     | ${'normal'}
+    ${'sansLight'}         | ${300}     | ${'normal'}
+  `(
+    'should apply provided font variant ',
+    ({ variant, fontWeight, fontStyle }) => {
+      render(
+        <InlineLink
+          to="/mundo/articles/ce42wzqr2mko"
+          text="Hello World!"
+          fontVariant={variant}
+        />,
+      );
+
+      expect(screen.getByText('Hello World!')).toHaveStyle({
+        color: '#222222',
+        'font-style': fontStyle,
+        'font-weight': fontWeight,
+      });
+    },
+  );
+
+  it('should not render client side link when CSR is enabled and location does not match regex', () => {
+    render(
+      <BrowserRouter>
+        <InlineLink to="https://google.com" text="Google" allowCSR />
+      </BrowserRouter>,
+    );
+    const anchorEl = screen.queryByTestId('client-side-link');
+
+    expect(anchorEl).toBeNull();
+    expect(screen.getByText('Google')).toBeInTheDocument();
+  });
+
+  it('should render link when CSR is enabled and provided link is split at first hashtag', () => {
```

Comment:
```suggestion
  it('should render link when CSR is enabled and link includes a hastag', () => {
```
Although the code tolerates multiple hashtags, its not really a valid url. Maybe just have this test checking it would with a hashtag url and another to test it works with a url without a hashtag?